project-base/docker/php-fpm-deployment/php-ini-overrides.ini

```diff
@@ -0,0 +1,49 @@
+date.timezone = ${TIMEZONE}
```

Comment:
Is this somehow different from the other php-ini-overrides?

Reply:
You are right, this file is not necessary anymore.